QUARKUS-3007.md

```diff
@@ -65,34 +58,31 @@ execution. We should consider moving the jobs to the same pipeline, managing its
 ### Impact on resources
 Product testing:
 * Catch-up on the test coverage on OCP to be same for aarch64 as it is for x86-64
-  * One additional OCP cluster for test services in AWS: 6x xlarge x86-64 machines in AWS
-  * EUS OCP in AWS: 6x xlarge Graviton machines, 4x large executors in Jenkins for each cell of JDK/OCP version matrix
+  * One OCP cluster for test services in AWS: 6x xlarge x86-64 machines in AWS
 * Certification of RHBQ native mode on OCP on aarch64
   * medium executor for orchestration
-  * 8x xlarge executors added from Beaker, 2 hours execution time
+  * 9x xlarge executors added from Beaker, 2 hours execution time
   * ideally, we will be using the same clusters as the other OCP jobs do, but we need at least
     * 1x test cluster - 6x xlarge Graviton machines in AWS
     * 1x test service cluster - 6x xlarge x86-64 machines in AWS
 * Certification of both the JVM and native modes in bare metal scenarios on aarch64
-  * `startstop-ts-code-quarkus` - 1x large aarch64 machine from Beaker per tested JDK, minutes to tens of minutes 
-    execution time
-  * `startstop-ts-special-chars` - 1x large aarch64 machine from Beakerper tested JDK, minutes to tens of minutes 
-    execution time
   * bare metal test suite
-    * JVM mode - 8x large aarch64 machine from Beaker per tested JDK, 1-2 hours execution time
-    * native mode - 8x xlarge aarch64 machine from Beaker, 2 hours execution time
+    * JVM mode - 9x large aarch64 machine from Beaker per tested JDK, 1-2 hours execution time
+    * native mode - 9x xlarge aarch64 machine from Beaker, 2 hours execution time
   * Quarkus Quickstarts
     * JVM mode - 1x large aarch64 machine fom Beaker per tested JDK, 1 hour execution time
-    * native mode - 1x xlarge aarch64 machine form Beaker, 2 hours execution time
+    * native mode - 1x xlarge aarch64 machine form Beaker, 5 hours execution time
 * Estimation of increase in time requirements
-  * For machine time, parallelism in execution should be established. The spin up of beaker machines does not take that
-    long in trials, but we don't have any significantly large sample size yet to say how it will look in long term.
+  * Beaker and OCP provisioning run in parallel and complete within 2 hours.
+  * Test jobs execute in parallel with x86-64 coverage on different machine quota.
+  * Running in parallel with x86-64 coverage, the pipeline should finish within 10 hours.
   * For people/investigation time, this depends on how different JDK and test services act on aarch64. So far, 
     significant functional bugs were not present on aarch64. All the issues historically were related to either missing
-    test services or missing productized dependencies. For native applications, we have no baseline to base our
-    estimates upon.
+    test services or missing productized dependencies. 
 
```

Comment:
Can you pls add overall time impact? So people do not need to sum the times mentioned above.

To me the implication is that the execution takes basically one work day. Investigation may trigger additional day or more, depending on the distribution of the failures and their root cause.

Reply:
I think I've added the execution estimation here - https://github.com/quarkus-qe/quarkus-test-plans/pull/189/files#diff-a60880627ee368f2d459af8d31ad2032f9e95d54958b619d3b73887a746efbdfR78. Basically it doesn't stretch the execution of extended platforms testing pipeline, since it runs in parallel on different infra, but it can run for a while.

Actually, I will remove the line about investigation. This here is the same as for regular bare-metal/OCP testing. There's nothing different here, and I don't think there's a useful estimation to be made here independent on the builds.